pallets/ddc-staking/src/lib.rs

```diff
@@ -227,6 +245,83 @@ pub mod pallet {
 	#[pallet::getter(fn current_era)]
 	pub type CurrentEra<T> = StorageValue<_, EraIndex>;
 
+	#[pallet::genesis_config]
```

Comment:
why do we need genesis config for ddc staking?

Reply:
Runtime mock uses genesis config in tests fixture. We use it here:

https://github.com/Cerebellum-Network/blockchain-node/blob/8911b9da57ed7c22db0f6dc455bc25cfe96fe97a/pallets/ddc-staking/src/mock.rs#L210

Also, conventional staking uses it in the same place, looks like it is a common practice: [paritytech/substrate/frame/staking/src/mock.rs#L504](https://github.com/paritytech/substrate/blob/dfd82860feb5da6f74ca2aa04414d9cebe310204/frame/staking/src/mock.rs#L504).